src/ansible/group_vars/all

```diff
@@ -56,3 +56,13 @@ user: {
     password: Secret123
   }
 }
+
+free_ipa_packages: {
+    server: [ freeipa-server, freeipa-server-dns, freeipa-server-trust-ad ],
+    client: [ freeipa-client, ]
+}
+
+ipa_packages: {
+    server: [ ipa-server-dns, ipa-server, ipa-server-trust-ad ],
+    client: [ ipa-client, ]
+}
```

Comment:
You are missing an empty line at the end of this file

Reply:
Added.